exercises/04_power/power_test.go

```diff
@@ -8,8 +8,20 @@ func TestPow(t *testing.T) {
 		x, y uint
 		want uint
 	}{
-		// Test cases.
+		{1, 1, 1},
```

Comment:
A good idea is to add tests where `x` is negative.

Reply:
And also both `x` and `y` are `0`